bench/src/bulk.rs

```diff
@@ -116,7 +117,14 @@ async fn client(server_addr: SocketAddr, server_cert: quinn::Certificate, opt: O
         .bind(&SocketAddr::new(IpAddr::V6(Ipv6Addr::LOCALHOST), 0))
         .unwrap();
 
-    let mut client_config = quinn::ClientConfigBuilder::default();
+    let mut config = quinn::ClientConfig::default();
+    let crypto_config: &mut rustls::ClientConfig =
+        Arc::get_mut(&mut config.crypto).context("retrieving crypto config failed")?;
```

Comment:
I'd just unwrap here, since it's trivially infallible.

Reply:
Agreed. We can also probably do without the type annotation here?